ui/src/app/profile-montior/profile-montior.component.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
This file(s) and its parent have a spelling errors.

Reply:
https://github.com/slaclab/slicops/pull/11/commits/748113c3a7fe89f9a37ac4556647751ff8c4e9b5